aviary/docs/user_guide/aviary_commands.ipynb

```diff
@@ -325,11 +325,11 @@
     "\n",
     "Example usage:\n",
     "```\n",
-    "`aviary convert_engine -f GASP PropFan.map PropFan.prop` Convert a GASP based propeller map\n",
-    "`aviary convert_engine -f GASP general_aviation.map general_aviation.prop` Convert a GASP based propeller map\n",
+    "`aviary convert_engine -f GASP PropFan.map [PropFan.prop]` Convert a GASP based propeller map\n",
+    "`aviary convert_engine -f GASP general_aviation.map [general_aviation.prop]` Convert a GASP based propeller map\n",
```

Comment:
Assuming I understand how the command works:
```suggestion
    "`aviary convert_engine -f GASP PropFan.map PropFan.prop` Convert a GASP based propeller map\n",
    "`aviary convert_engine -f GASP general_aviation.map` Convert a GASP based propeller map\n",
```

Reply:
Okay.